versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md

```diff
@@ -224,6 +224,15 @@ Sequelize also lets you pass credentials in URL format:
 const sequelize = new Sequelize('oracle://user:pass@hostname:port/servicename');
 ```
 
+You can pass an Easy Connect String, a Net Service Name, or a Connect Descriptor to the Sequelize constructor using `dialectOptions.connectString`:
+
+```js
+const sequelize = new Sequelize({dialect: 'oracle', username: 'user', password: 'password', dialectOptions: {connectString: 'inst1'}});
+```
+Note that the `host` and `port` will be overwritten to values available in connectString.
```

Comment:
Are they really written to?  Or did you mean they are "overridden by", i.e ignored?  Also should 'database' be mentioned here?

Reply:
You should mention it somehow, since it will be confusing if they set a value but it is ignored.

Reply:
I have updated the sentence to also include `database`.